vue/components/ui/organisms/details/details.vue

```diff
@@ -9,7 +9,15 @@
         <container-ripe class="loading" v-if="isLoading">
             <template v-slot:header>
                 <slot name="title" v-if="isLoaded">
-                    <title-ripe>{{ invalid ? invalidTitle : title }}</title-ripe>
+                    <breadcrumbs
+                        v-bind:font-size="breadcrumbsFontSize"
+                        v-bind:separator="breadcrumbsSeparator"
+                        v-bind:breadcrumbs="title"
+                        v-if="title && Array.isArray(title)"
+                    />
+                    <title-ripe v-else>
+                        {{ invalid ? invalidTitle : title }}
+                    </title-ripe>

```

Comment:
Apply this strategy "v-if=array + v-else" to `vue/components/ui/atoms/container/container.vue`